controller/space_test.go

```diff
@@ -356,3 +371,92 @@ func minimumRequiredUpdateSpace() *app.UpdateSpacePayload {
 		},
 	}
 }
+
+func initTestIdenity(db *gorm.DB) account.Identity {
+	identityRepo := account.NewIdentityRepository(db)
+	testIdentity := getTestIdentity()
+	err := createIdentity(context.Background(), testIdentity, identityRepo)
+	if err != nil {
+		panic(err.Error())
+	}
+	return *testIdentity
+}
+
+func initTestIdenity2(db *gorm.DB) account.Identity {
+	identityRepo := account.NewIdentityRepository(db)
+	testIdentity := getTestIdentity2()
+	err := createIdentity(context.Background(), testIdentity, identityRepo)
+	if err != nil {
+		panic(err.Error())
+	}
+	return *testIdentity
+}
+
+func createIdentity(ctx context.Context, identity *account.Identity, identityRepo account.IdentityRepository) error {
+	_, err := identityRepo.Load(ctx, identity.ID)
+	if err != nil {
+		user := account.User{
+			Email:    uuid.NewV4().String() + "@domain.com",
+			FullName: identity.Username,
+		}
+		identity.User = user
+		identity.UserID = account.NullUUID{UUID: user.ID, Valid: true}
+		err := identityRepo.Create(ctx, identity)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+// getTestIdentity returns an identity which represents an existing Keycloak test identityonly
+func getTestIdentity() *account.Identity {
```

Comment:
This stuff is binding the test cases that is defined as ```resource.Require(t, resource.Database)``` to have access to sso.almighty.io as well, right?

We need a way to run these without having a KC available, and we need to introduce another resource requirement level for things that require external resources like .e.g. KC.

Else we can't run any tests offline and the test suite will slow to a hault and start to fail more randomly due to network issues.